crates/sui-types/src/object.rs

```diff
@@ -57,28 +57,6 @@ pub struct MoveObject {
 /// Index marking the end of the object's ID + the beginning of its version
 pub const ID_END_INDEX: usize = ObjectID::LENGTH;
 
-/// Different schemes for converting a Move value into a structured representation
-#[derive(Eq, PartialEq, Debug, Clone, Deserialize, Serialize, Hash)]
-pub struct ObjectFormatOptions {
```

Comment:
This wound up being able to be removed completely as it's an old appendage for formatting Move values in JSON. However, it was only ever called with `include_types` set to true. Since that is now the only way we generate it, it made sense to just get rid of this completely as it's no longer useful or meaningful.